src/script/ptl-tool.py

```diff
@@ -477,8 +477,9 @@ def build_branch(args):
         if args.qa_tags:
             custom_fields.append({'id': REDMINE_CUSTOM_FIELD_ID_QA_TAGS, 'value': args.qa_tags})
 
-        G.git.push(CI_REMOTE_URL, branch) # for shaman
-        G.git.push(CI_REMOTE_URL, tag.name) # for archival
+        if not args.no_build:
+            G.git.push(CI_REMOTE_URL, branch) # for shaman
+            G.git.push(CI_REMOTE_URL, tag.name) # for archival
```

Comment:
In what circumstances do you find this useful?

Reply:
When I need to test a bunch of QA PRs urgently, I can do so by merging these PRs on a new branch and pass it to `--suite-branch` option while I use builds from shaman for recent version of main branch.

I've been doing this today lots of times. Venky wanted me to have a tracker ticket for future reference.

This change will be handy in future too when QA PRs needs to be collected and tested against main, squid, reef, etc.

Reply:
I've mentioned this in Slack chat and commit message too. Let me know if it needs to be updaed.

Reply:
It's a fine use-case. I suggest adding it to the top-level README comments as an example. Be aware however that other changes in the qa suite on `--base main` (the default) may unexpectedly change the `qa/` suite in ways that surprise.

Reply:
Done.